pkg/dockercompose/dockercompose.go

```diff
@@ -81,6 +81,9 @@ func Load(composePath string, overridePaths, services []string) (types.Config, e
 
 		// Skip validation so that the loader doesn't error on non-v3 files.
 		withSkipValidation,
+
+		// Skip consistency check since it's broken for volumes that are just a path.
+		withSkipConsistency,
```

Comment:
Is this only broken in the new version? I would think we would have seen this before.

Reply:
They didn't add consistency checks until recently